apiary.apib

```diff
@@ -267,6 +268,35 @@ Token authentication endpoint.
 
     [Token][]
     
++ Response 400
+
+    In this case is required second factor authentication.
```

Comment:
Možná bych to napsal spíš _In this case second factor authentication is required._
Ale je to jen pocitově... nejsem nějakej velkej angličtinář. :-)

Reply:
Rekl bych ze ta forma co pise Vita je beznejsi

Reply:
Dal som to tak :) Dik